src/server/raft_client.rs

```diff
@@ -930,29 +936,44 @@ where
         }
     }
 
+    pub fn delay_flush(&mut self) {
+        if !self.need_flush() {
+            return;
+        }
+        let now = TiInstant::now_coarse();
+        if now > self.last_flush_time + self.delay_flush {
```

Comment:
Why not use a timer in future executor instead?

Reply:
Ok. I will propose another PR to fix this. I remove these codes.